apps/api/src/app/workflows/usecases/create-notification-template/create-notification-template.usecase.ts

```diff
@@ -263,6 +278,18 @@ export class CreateNotificationTemplate {
         _organizationId: command.organizationId,
         name: blueprintNotificationGroup.name,
       });
+
+      await this.createChange.execute(
+        CreateChangeCommand.create({
+          item: group,
+          environmentId: command.environmentId,
+          organizationId: command.organizationId,
+          userId: command.userId,
+          type: ChangeEntityTypeEnum.NOTIFICATION_GROUP,
+          parentChangeId: parentChangeId,
```

Comment:
```suggestion
          parentChangeId,
```
@djabarovgeorge doesn't setting the parent here will make it into the same row? Like, it won't show as an independent change?

Reply:
Yes, and it is the desired behavior i think we should merge all the possible changes into one workflow change in order to reduce potential issues like we had in the past
Do you have another opinion on the matter?

Reply:
Its just not the current behavior we have for notification group. They are showing separately ->notification group change can be promoted as itself. But if you promote the notification template change  it promotes both changes